mods/ra/maps/allies-01/rules.yaml

```diff
@@ -22,6 +22,9 @@ TRAN.Extraction:
 	RenderSprites:
 		Image: tran
 	Interactable:
+	WithCargoHatchAnimation:
+	Aircraft:
+		InitialFacing: 0
```

Comment:
In my opinion we should add this to the campaign rules, can't think of a reason why any singleplayer mission wouldn't want this.

Same for TD, btw.

Reply:
Expanded this from allies01 only to all campaign missions.